src/agent_IR_RaymondWang_LinYang.py

```diff
@@ -0,0 +1,98 @@
+#trying
+""" Information Ratio """
+import argparse
+import numpy as np
+from collections import deque
+from pedlar.agent import Agent
+import time as tm
+
+
+
+class IRAgent(Agent):
+    name = "IR"
+    def __init__(self, verbose=False, **kwargs):
+        self.verbose=verbose
+        self.slow = deque(maxlen=128)
+        self.fast = deque(maxlen=32)
+        self.last_order = None
+        self.price_list=[]
+        self.std=0
+        self.total_profit_list=[]
+        self.total_profit=0
+        self.order_in_progress=False
+        super().__init__(**kwargs)
+
+
+    # print order when placing an order
+    def on_order(self, order):
+        """On order handler."""
+        print('----------------------------------------------')
+        print("ORDER: ", order)
```

Comment:
remove the print("-------------")

add:
if self.verbose:
    print(f"Order: {order}")

Reply:
if I add those two lines, shall I add a verbose input in the def() as well?

Reply:
verbose input isnt required since self.verbose is a property of the class and can be access or modified anywhere in the class.